src/proxy-api/proxy-api.controller.ts

```diff
@@ -171,6 +172,21 @@ export class ProxyApiController {
     return this.proxyApi.getJiraUsersByQuery(queries.query, queries.startAt, queries.maxResults);
   }
 
+  /**
+   * @GET (controller) api/projects/versions
+   * @description Returns all versions in a project
+   * @return {string} 'JSON' - Returns the valid versions for a project.
+   */
+  @ApiOkResponse({
+    description: 'List project versions',
+  })
+  @Get('projects/versions')
+  async getJiraProjectVersions(
+    @Query() queries: SearchProjectVersionsQueryDTO,
+  ): Promise<any> {
```

Comment:
Please specify what is the response instead of Promise<any>

Reply:
Specified the type as FixVersion